core/server/common/src/main/java/alluxio/ServerRpcUtils.java

```diff
@@ -33,8 +37,11 @@
  * For each of these, if failureOk is set, non-fatal errors will only be logged at the DEBUG
  * level and failure metrics will not be recorded.
  */
-public final class RpcUtils {
-  private RpcUtils() {} // prevent instantiation
+public final class ServerRpcUtils {
```

Comment:
Renamed `RpcUtils` to `ServerRpcUtils` because this is the server side logic (various `XXXServiceHandler` and `XXXReadHandler` classes). This class is in `alluxio-core-server-common` so I think it should be made obvious that this is the server RPC utilities.

Reply:
can we have a separate PR for the rename and leave only real change in this PR?
This is because this PR has 600+lines of change and most of them are because of rename. Ideally, let's have incremental changes once a time to reduce review complexity.

Reply:
I began to regret immediately after I realized how much I updated a few commits later TAT
Reverted